docs/UserGuide.md

```diff
@@ -285,6 +291,7 @@ Action | Format, Examples
 **Clear** | `clear`
 **Delete** | `delete INDEX`<br> e.g., `delete 3`
 **Delete Event** | `deleteEvent INDEX`<br> e.g., `deleteEvent 2`
+**Delete Event** | `deleteEvent INDEX`<br> e.g., `deleteEvent 2`
```

Comment:
Duplicate.

Reply:
Thanks for pointing it out. Have made the edit.